src/pages/edit_resource/contributors_step/ContributorFields.tsx

```diff
@@ -256,11 +251,36 @@ const ContributorFields: FC<ContributorFieldsProps> = ({ setAllChangesSaved }) =
                           <HelperTextPopover
```

Comment:
kanskje vi burde flyttet hjelpe-iconet til overskriften for Opphaver og Bidragsyter - istedenfor å legge den bak første innslag ?

Reply:
Fikset